drivers/audio/wm8904.c

```diff
@@ -264,17 +264,17 @@ static int wm8904_in_update(
 static int wm8904_in_volume_config(const struct device *dev, audio_channel_t channel, int volume)
 {
```

Comment:
This is a bugfix unrelated to the original goal of this PR. Please create another PR for it. Also please fix warnings like this:

![image](https://github.com/user-attachments/assets/40d187d6-6be9-4224-8793-21ccdeed01ba)